### ISSUE TITLE
 multi person

### ISSUE THREAD

**Issue by sunlinlin-aragon** [rawalkhirodkar/MIPNet#10]
Hello, do you support multi person key point identification? I don't see where it was set up

**Comment by rawalkhirodkar** [comment]
Yes, the code can regress multiple persons from a single bounding box. 
Please refer the testing code at [https://github.com/rawalkhirodkar/MIPNet/blob/main/tools/lambda/test_lambda.py](link)
